.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "8"
+  - "10"
+git:
+  depth: 5
+script:
+  - yarn build
+  - yarn lint
+  - yarn test-travis
```

Comment:
lint / test-travis scripts from package.json don't actually exist, so travis is currently failing. If you want, you can remove the test and I can send you a PR for setting up BS